src/Serilog.Sinks.RabbitMQ/Serilog.Sinks.RabbitMQ.csproj

```diff
@@ -22,6 +21,22 @@
     <PackageReference Include="MinVer" Version="5.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup  Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="2.1.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="7.0.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="8.0.0" />
```

Comment:
Is it on purpose that you don't use 8.0.3?

Reply:
Yes. I work in different repos. Some bump dependencies frequently, even patches. Some tend to reference only minimum requered for ages. I would not say there is the best strategy for any case.

Reply:
I think that updating the package with dependabot will ensure our package runs on the newest patches. That's the only argument I can think of.

Reply:
So? 8.0.0 or 8.0.3 ? We can merge with 8.0.0 , publish v7 release and then bump versions for v7.0.1.

Reply:
Sure